### PR TITLE
Tweak documentation for tools/[azure|docker]

### PR DESCRIPTION
**Pull request by sobi3ch**
Minor tweaks in azure and docker tools sections

### CODE REVIEW

File: tools/docker/hub/Dockerfile [hunk 1]
```diff
@@ -13,5 +13,3 @@ RUN set -xe; \
 # Stage 2
 FROM ${BASE_IMAGE}
 COPY --from=0 /work/implementations/elixir/ockam/ockam_hub/_build/prod/rel/ockam_hub /opt/ockam_hub
-ENTRYPOINT ["/opt/ockam_hub/bin/ockam_hub"]
-CMD ["start"]
```
Comment (mrinalwadhwa) [inline tools/docker/hub/Dockerfile:17]: I think we need to keep this as the default entrypoint and command ... the azure container instance will run as long as the default command blocks. If we don't have this then the entrypoint of the base image will have to used some how to define a blocking command.

**Comment by mrinalwadhwa** [comment]
Note that the image name has changed to `ockam/hub:latest`
https://github.com/ockam-network/ockam/tree/303b0145b0b07e671a3a449c368a028104cbc359/tools/docker#hub

The references to the image from azure.json and README should change from `ockam/ockam_hub:latest` to `ockam/hub:latest`

**Comment by mrinalwadhwa** [comment]
Also while you're amending cd112be could you also change the commit message to be in the imperative mood, like the other ones... so maybe something like `build: remove azure instance command override`

The commits marked as `chore:` can, I think, all be marked as `build:` instead as they affect the build of our infra.

**Comment by mrinalwadhwa** [comment]
@sobi3ch Yay !! on your first PR 🎉 
left some notes above for some minor improvements.

File: tools/docker/hub/Dockerfile (hunk 1 above)
Comment (sobi3ch) [reply tools/docker/hub/Dockerfile:17]: done

**Comment by sobi3ch** [comment]
> README should change from `ockam/ockam_hub:latest` to `ockam/hub:latest`

yeap realize this already :) thx

**Comment by sobi3ch** [comment]
chore -> build: done
imperative mood: done

**Comment by mrinalwadhwa** [comment]
The changes here look ready to merge, a couple of quick commit impoments:

1. could you squash d1474756b175bfdfd513bd62f6428c8a4f00dfc8  into 42b5b81f257c4f328c624578246516a48159cc8b 
```
git rebase -i HEAD~2
```
and then `fixup` d1474756b175bfdfd513bd62f6428c8a4f00dfc8

2. could you change the commit message for a5320f20f3e81034bff812e1efb4f45933d640b1 to describe what is changing -
something like `build: use ockam/hub image in azure container manifest`

**Pull request closed without merging**